DMTN-266.tex

```diff
@@ -48,6 +49,178 @@
 
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
+\section{Introduction}
+The goal of the astrometric fitting in the LSST science pipeline is to obtain precise alignment of individual exposures relative to each other along with accurate alignment with an external reference catalog. This alignment is necessary to produce accurate coadded images downstream in the pipeline. It also is necessary for obtaining precise measurements of all objects and for measuring proper motion and parallax. In particular, the astrometric calibration is necessary to attain the design requirements enumerated in the LSST System Science Requirements Document (LPM-17) and related documents.
+
+\section{Astrometry in the science pipeline}
+\subsection{Preliminary Astrometric Solutions}
+An initial estimate of the World Coordinate System (WCS) mapping is made using the camera model. This camera model is currently assumed to be static (see Sections~\ref{sec:DetMod} and~\ref{sec:out} for discussion of improvements to this approach). In single-frame processing, this WCS is further refined by matching point sources to an external catalog and fitting a mapping per detector for each exposure. This fit is done by \texttt{AstrometryTask} as part of \texttt{calibrateTask} or \texttt{calibrateImageTask}, (the latter will be the long-term place for this step). For this mapping, the map can be configured to be either a TAN-SIP WCS (\citealt{Shupe2005}) (implemented in \texttt{FitTanSipWcsTask}), or the input camera model followed by an affine transformation (\texttt{FitAffineWcsTask}). The latter is preferred from a perspective of speed and robustness, but will give poor results if the input camera model is not sufficiently accurate. \texttt{FitTanSipWcsTask} is used for most instruments in the DRP pipeline, whereas \texttt{FitAffineWcsTask} is used for the AP pipeline.
+
+The reference catalog currently used is from Gaia DR3 (\citealt{Gaia2023}), which covers the whole sky, with a limiting magnitude of about $21$ in g-band. The accuracy of this fit is limited by the number of sources in an exposure that can be matched to the reference catalog and by the uncertainty on the centroids of the sources in both the science and reference catalogs. However, after this initial fit, the WCS is accurate to within $1$ arcsecond, which is sufficient for associating isolated sources across visits.
```

Comment:
The approximate limiting magnitude for Gaia is much closer to 21 in the r-band, not the g-band.  The fact that Gaia named their wide band "G" (for Gaia) has confused many observers who read this as Johnson G.  Although not matched at all, the peak of the Gaia response is very close to the LSST r band.